db/profile_repository.go

```diff
@@ -1,14 +1,18 @@
 package db
 
 import (
+	pb "github.com/Kotlang/authGo/generated"
 	"github.com/Kotlang/authGo/models"
+	"github.com/SaiNageswarS/go-api-boot/logger"
 	"github.com/SaiNageswarS/go-api-boot/odm"
 	"go.mongodb.org/mongo-driver/bson"
+	"go.uber.org/zap"
 )
 
 type ProfileRepositoryInterface interface {
 	odm.BootRepository[models.ProfileModel]
 	FindByIds(ids []string) (chan []models.ProfileModel, chan error)
+	GetUserIds(userfilters *pb.Userfilters, PageSize, PageNumber int64) []models.ProfileModel
```

Comment:
Improve naming. Name the function GetMultipleProfiles/users. It is not fetching userIds, a good name should give the context of what the function does.